blueocean-core-js/src/js/utils.js

```diff
@@ -21,6 +21,7 @@ const cleanSlashes = (url: string) => {
 export default {
     cleanSlashes,
     clone(obj: Object) {
+        if (!obj) return obj;
```

Comment:
The intent of the method is to clone, so I am not sure what else would be appropriate to return. Empty object doesn't quite seem right, can't think of anything else TBH.